04LessonFour/in-class-project/index.html

```diff
@@ -1,10 +1,65 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>The Daily Planet</title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
-
-</body>
-</html>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Intel</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <header>
+        <h1>Intel</h1>
+        <nav>
```

Comment:
Great work, Tu! The code is organized, correct and nested well. I would only create a little more vertical space between elements, but that is subjective.